<launch>
	
	<!-- load reward map configurations from YAML file to parameter server -->
	<rosparam file="$(find terrain_server)/config/reward_map.yaml" command="load"/>
	
	<node pkg="terrain_server" type="reward_map_server" name="reward_map" output="screen">
		<remap from="reward_map" to="/reward_map" />
		<!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
		<param name="world_frame" type="string" value="odom" />
		<!-- Base frame of the robot -->
		<param name="base_frame" type="string" value="base_link" />
	</node>
	
</launch>